sdk/keyvault/azure-keyvault-certificates/src/main/java/com/azure/security/keyvault/certificates/CertificateServiceVersion.java

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.security.keyvault.certificates;
+
+/**
+ * The versions of Azure Key Vault supported by this client library.
+ */
+public enum CertificateServiceVersion implements com.azure.core.http.ServiceVersion {
+    V7_0("7.0");
+
+    private final String version;
+
+    CertificateServiceVersion(String version) {
```

Comment:
Make this constructor private

Reply:
Aren't enum constructors private or package-private by default? Gives me a redundant access modifier.